lib/navigation.php

```diff
@@ -50,6 +50,9 @@ function gutenberg_register_navigation_post_type() {
 			'editor',
 			'revisions',
 		),
+		'capabilities'          => array(
+			'create_posts' => 'edit_theme_options',
+		),
```

Comment:
```suggestion
		'capabilities'          => array(
				// Meta Capabilities.
				'edit_post'              => 'edit_post',
				'read_post'              => 'read_post',
				'delete_post'            => 'delete_post',
				// Primitive Capabilities.
				'edit_posts'             => 'edit_posts',
				'edit_others_posts'      => 'edit_theme_options',
				'delete_posts'           => 'edit_theme_options',
				'publish_posts'          => 'edit_theme_options',
				'read_private_posts'     => 'edit_theme_options',
				'read'                   => 'read',
				'delete_private_posts'   => 'edit_theme_options',
				'delete_published_posts' => 'edit_theme_options',
				'delete_others_posts'    => 'edit_theme_options',
				'edit_private_posts'     => 'edit_theme_options',
				'edit_published_posts'   => 'edit_theme_options',
			),
```

@getdave Can you try these values and see if they work for you? I think this line is what was the issue `'edit_posts'             => 'edit_posts',`

Reply:
After more testing, the above does not work.

Reply:
Can you try the following 

```
		'capabilities'          => array(
			'edit_others_posts'      => 'edit_theme_options',
			'delete_posts'           => 'edit_theme_options',
			'publish_posts'          => 'edit_theme_options',
			'create_posts'           => 'edit_theme_options',
			'read_private_posts'     => 'edit_theme_options',
			'delete_private_posts'   => 'edit_theme_options',
			'delete_published_posts' => 'edit_theme_options',
			'delete_others_posts'    => 'edit_theme_options',
			'edit_private_posts'     => 'edit_theme_options',
			'edit_published_posts'   => 'edit_theme_options',
		),
```

Reply:
Ok that _seems_ to work so far in my testing.